misc/all_tests.conf

```diff
@@ -3,3 +3,4 @@ include misc/host_tests.conf
 
 # All contributed modules.
 include subset/switches/build.conf
+include subset/mac/build.conf
```

Comment:
"mac" is too vague -- as mentioned in the email, need something a bit more categorical. "macoui" is too specific. Out of this specific test, what's the category it falls into?  Is it identity tests?  l2 networking tests?  Doesn't need to be absolute, but should at least be able to encompass one other test that we think might happen eventually. And not be so broad that it's totally generic.

Reply:
This comment has not been addressed.

Reply:
changed "mac" to "connection"